.circleci/ds005_outputs.txt

```diff
@@ -14,6 +14,8 @@ fmriprep/sub-01/anat/sub-01_T1w_midthickness.R.surf.gii
 fmriprep/sub-01/anat/sub-01_T1w_pial.L.surf.gii
 fmriprep/sub-01/anat/sub-01_T1w_pial.R.surf.gii
 fmriprep/sub-01/anat/sub-01_T1w_preproc.nii.gz
+fmriprep/sub-01/anat/sub-01_T1w_rois-fsaparcaseg.nii.gz
+fmriprep/sub-01/anat/sub-01_T1w_rois-fsaseg.nii.gz
```

Comment:
What's the advantage of including `aseg` in addition to `aparc+aseg`? `aseg` is just `aparc+aseg` with all cortical labels collapsed into "gray matter".

It's not going to be a huge cost in space, so that's fine.

Also, the [derivatives spec](https://docs.google.com/document/d/1Wwc4A6Mow4ZPPszDIWfCUCRNstn7d_zzaWPcfcHmgI4/edit#heading=h.9146wuepclkt) says that the type of these files should be `roi`, so they should end with `_roi.nii.gz`. And instead of `rois-`, I would use `label-`, to be consistent with that spec.

Finally, I think `aseg` and `aparcaseg` are just as unambiguous as `fsaseg` and `fsaparcaseg` and a little easier to read.